storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,76 @@
+.. _page_tracking:
+
+====================================
+Page Tracking for Incremental Backup 
+====================================
+
+As of 8.0.18, the MySQL supports the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup the |mysqlbackup| looks for pages that have been modified since the last backup in the InnoDB data files and copies only changed pages. Since there is no need to scan all the pages in the database, the page tracking makes the incremental backup faster if the number of changed pages is not big.
+
+Our tests show that if |1%| of data has been changed after the full backup of |100 GB|, the incremental backup takes only |30 seconds|. In comparison, without page tracking, the incremental backup takes |5 minutes|.
+
+If the number of changed pages increases, the performance reduces. Our tests show, that with page tracking the incremental backup performs better than full scan up to |50%| the size of the server*. 
+
+.. note::
+
+   The result depends on the type of workload. For example, you either insert new pages or randomly update different pages in the database.
+
+Prerequisites
+-------------
+
+To start using the page tracking functionality, do the following steps:
+
+1. Install the |mysqlbackup| component and enable it on the server: 
+
+  .. code-block:: bash
+
+     INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the |mysqlbackup| component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE  component_urn='file://component_mysqlbackup';
+
+Usage
+-----
+
+After the |component_mysqlbackup| is loaded and active on the server, enable the page tracking functionality for the full and incremental backups using the following option:  
+
+``--page-tracking``
+
+It serves the dual purpose:
+
+* Reset page tracking at the start of the backup so that the next incremental backup can use page tracking.
+
+* Use page tracking for incremental backup if page tracking data is available from the backup start checkpoint LSN.
+
+Once the page tracking is enabled, the next incremental backup will use page tracking to find changed pages.
+
+You can check the LSN value starting from which changed pages have been tracked using the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_get_start_lsn();
+
+To stop page tracking use the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(false);
+
+.. note::
+
+   To use the aforementioned options for page tracking, run the ``BACKUP_ADMIN`` privilege.
```

Comment:
This information (privileges) should be in the beginning instead of after the user has tried the function.

Reply:
Fixed